pkg/liqonet/utils_test.go

```diff
@@ -0,0 +1,99 @@
+package liqonet_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/liqotech/liqo/pkg/liqonet"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Liqonet", func() {
+
+	ns := "default"
```

Comment:
According to the Kingko paradigm, the variables should be first declared and then instantiated in a beforeEach block. This approach is required to be sure to have the variables always clean and not affected by previous test cases.